tests/e2e/modify_volume.go

```diff
@@ -123,8 +123,11 @@ var _ = Describe("[ebs-csi-e2e] [single-az] [modify-volume] Modifying a PVC", fu
 	for testName, modifyVolumeTest := range modifyVolumeTests {
 		modifyVolumeTest := modifyVolumeTest
 		Context(testName, func() {
-			It("will modify associated PV and EBS Volume", func() {
-				modifyVolumeTest.Run(cs, ns, ebsDriver)
+			It("will modify associated PV and EBS Volume via volume-modifier-for-k8s", func() {
+				modifyVolumeTest.Run(cs, ns, ebsDriver, testsuites.VolumeModifierForK8s)
```

Comment:
The vast majority (something like 70% - 80%) of such functions would be duplicated and performing the same task. Avoiding significant duplication of code (especially when it would be verbatim the same) is a common software engineering principle: https://en.wikipedia.org/wiki/Don%27t_repeat_yourself